python/hail/docs/tutorials/03-expressions.ipynb

```diff
@@ -293,9 +498,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# `show`, `take`, and `collect`\n",
+    "### `show`, `take`, and `collect`\n",
+    "\n",
+    "Although expressions with indices have no `value`, you can use `show` to print the first few values, or `take` and `collect` to localize values to Python. \n",
```

Comment:
Although expressions with indices do not have a single realizable value (calling the `value` method will fail), you can ...